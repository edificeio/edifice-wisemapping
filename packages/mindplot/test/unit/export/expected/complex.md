# PPM Plan

- Business Development 
- Backlog Management ( [link](https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit) )
- Freeform IT
- Client Project Management
- Governance & Executive
- Finance
- Administration
- Human Resources[^1] 
- Freeform Hosting
- Community Outreach
- R&D
	- Goals
	- Formulize
- Probono



[^1]: HR Vision: Freeform Solutions is successful at its mission, sustainable as an organization AND is a great place to work.HR Mission: To provide a positive HR service experience for applicants and employees, and  collaborate with departments to recruit, develop, support, and retain diverse and talented employees who are the key to Freeformâ€™s  reputation and success.
