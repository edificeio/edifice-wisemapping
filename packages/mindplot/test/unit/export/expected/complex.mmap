<?xml version="1.0" encoding="UTF-8"?><ap:Map xmlns:ap="http://schemas.mindjet.com/MindManager/Application/2003"><ap:OneTopic><ap:Topic OId="1W54nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="5W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Finance"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2064nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Governance &amp; Executive"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="3W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Administration"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1854nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Backlog Management"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/><ap:Hyperlink Url="https://docs.google.com/a/freeform.ca/drawings/d/1mrtkVAN3_XefJJCgfxw4Va6xk9TVDBKXDt_uzyIF4Us/edit"/></ap:Topic><ap:Topic OId="2044nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Client Project Management"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2684nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2694nezC90m8NYAi2fjQvw=="><ap:Text PlainText=""><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="Probono"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1054nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Freeform IT"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2474nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Community Outreach"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1544nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Human Resources"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="1654nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Freeform Hosting"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="4W54nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Business Development "><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2614nezC90m8NYAi2fjQvw=="><ap:SubTopics><ap:Topic OId="2634nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Goals"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic><ap:Topic OId="2644nezC90m8NYAi2fjQvw=="><ap:Text PlainText="Formulize"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="R&amp;D"><ap:Font Bold="true" Size="12"/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:Line"/></ap:Topic></ap:SubTopics><ap:Text PlainText="PPM Plan"><ap:Font/></ap:Text><ap:SubTopicShape SubTopicShape="urn:mindjet:RoundedRectangle&#10;                        "/><ap:SubTopicsShape SubTopicsAlignment="urn:mindjet:Center" SubTopicsGrowth="urn:mindjet:Horizontal" SubTopicsGrowthDirection="urn:mindjet:AutomaticHorizontal&#10;                        " VerticalDistanceBetweenSiblings="150"/></ap:Topic></ap:OneTopic><ap:Relationships/></ap:Map>