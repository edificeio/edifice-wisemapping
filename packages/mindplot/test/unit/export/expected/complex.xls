<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<?mso-application progid="Excel.Sheet"?><Workbook xmlns:x="urn:schemas-microsoft-com:office:excel" xmlns:ss="urn:schemas-microsoft-com:office:spreadsheet" xmlns:duss="urn:schemas-microsoft-com:office:dummyspreadsheet" xmlns:o="urn:schemas-microsoft-com:office:office" xmlns="urn:schemas-microsoft-com:office:spreadsheet">
<Styles>
<Style ss:ID="s16" ss:Name="attribute_cell">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
</Style>
<Style ss:ID="s17" ss:Name="attribute_header">
<Borders>
<Border ss:Position="Bottom" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Left" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Right" ss:LineStyle="Continuous" ss:Weight="1"/>
<Border ss:Position="Top" ss:LineStyle="Continuous" ss:Weight="1"/>
</Borders>
<Font ss:Bold="1"/>
</Style>
</Styles>
<Worksheet ss:Name="FreeMind Sheet">
<Table>
<Row>
<Cell ss:Index="1">
<Data ss:Type="String">PPM Plan</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Finance</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Governance &amp; Executive</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Administration</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Backlog Management</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Client Project Management</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Probono</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String"/>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Freeform IT</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Community Outreach</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Human Resources</Data>
<Comment>
<ss:Data xmlns="http://www.w3.org/TR/REC-html40">
<p/>
</ss:Data>
</Comment>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Freeform Hosting</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">Business Development </Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="2">
<Data ss:Type="String">R&amp;D</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Goals</Data>
</Cell>
</Row>
<Row>
<Cell ss:Index="3">
<Data ss:Type="String">Formulize</Data>
</Cell>
</Row>
</Table>
</Worksheet>
</Workbook>
