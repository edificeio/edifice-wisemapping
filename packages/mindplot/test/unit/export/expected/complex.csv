PPM Plan 
,
            Finance 
,
            Governance & Executive 
,
            Administration 
,
            Backlog Management 
,
            Client Project Management 
,
            Probono 
,
            ,
             
,
            Freeform IT 
,
            Community Outreach 
,
            Human Resources 
,
            Freeform Hosting 
,
            Business Development  
,
            R&D 
,
            ,
            Goals 
,
            ,
            Formulize 
